tools/BUILD.bazel

```diff
@@ -1,28 +1,5 @@
 package(default_visibility = ["//visibility:public"])
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
-load("//tools:defaults.bzl", "ts_library")
-
-ts_library(
-    name = "sass_bundle_lib",
-    srcs = ["sass-bundle.ts"],
```

Comment:
How are we handling the Sass bundling and copying now that this is removed?

Are we still using the `scss-bundle` dependency?

Reply:
rules_nodejs has a new feature that lets you use a npm package from node_modules without setting up extra rules

https://github.com/angular/components/blob/29be7686efa55d9027db08c50717789b23021a78/src/material/BUILD.bazel#L24-L34

Reply:
Ah, that's nice!